torch/export/dynamic_shapes.py

```diff
@@ -1191,3 +1193,280 @@ def apply_fixes(path, dim, dummy):
         return dim  # unchanged dim
 
     return _tree_map_with_path(apply_fixes, dynamic_shapes, dynamic_shapes)
+
```

Comment:
nit: what if we moved these functions to _export/serde? 

also suggestion for the names, can we suffix the functions with `_dynamic_shapes`? Like, `dump_dynamic_shapes`